src/components/Button.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button(props) {
+  const { value } = props;
```

Comment:
### [Optional]
For good naming convention kindly use `buttonName`  instead of `value`